### PR TITLE
Add a code-review guide

### PR DESCRIPTION
**Pull request by artasparks**
This PR adds a code-review guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,12 +38,18 @@ If you're new to the Golang (the Go programing language), then check out our
 
 ## Getting Started with Git
 
-If you're new to using a git or Gihub,check out our
+If you're new to using a git or Gihub, check out our
```
Comment (artasparks) [inline CONTRIBUTING.md:41]: This needs more color.

Comment (artasparks) [reply CONTRIBUTING.md:41]: I don't think so.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,12 +38,18 @@ If you're new to the Golang (the Go programing language), then check out our
...
 [getting started with git guide](/gettingstarted/git.md).
 
+## Code Review
+
+We require submissions to all core Otrego repositories have code-review. Check
+out our [code review guide](/gettingstarted/codereview.md).
+
 ## Getting Started with Katago
 
 Some what we do involves working with the open source go AI
```
Comment (artasparks) [inline CONTRIBUTING.md:51]: srchaoeusrcaohe usrch

Comment (artasparks) [inline CONTRIBUTING.md:51]: zork

File: README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # Otrego Community
 
-Otrego is a community dedicate to working together to making better,
-open-source software for the
-<a href="https://en.wikipedia.org/wiki/Go_(game)">board game Go</a>, also known as igo
-(囲碁), baduk (바둑), weiqi (围棋).
+Otrego is a community of programmers and go-players dedicated to working
+together to making a community around making better, open-source software for
+the <a href="https://en.wikipedia.org/wiki/Go_(game)">board game Go</a>, also
+known as igo (囲碁), baduk (바둑), weiqi (围棋).
```
Comment (TamaBrian) [inline README.md]: known as igo (囲碁) in Japan, baduk (바둑) in Korea, weiqi (围棋) in China.

File: gettingstarted/codereview.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+perfection*. Even if you, as a code-reviewer, have *no comments* just having a
+second pair of eyes on code tends to improve code-quality.
+
+2.  **Code Reviews should be Quick**. Code reviewers should respond to the code
+    review in ~24 hours and aim to complete the code review in 2-3 days or
+    less, depending on the size of the code reveew. If a code-reviewer hasn't
+    responded on a Pull Request in 24 hours, it's OK to ping the code-reviewer.
+
+1.  **Focus on Understanding** Focus on understanding the change first. What is
+    the trying to do? Feel free to ask questions! Otrego is about collaboration
```
Comment (TamaBrian) [inline gettingstarted/codereview.md]: What is the [ ??? ] trying to do?

Comment (nelsonjhk) [reply gettingstarted/codereview.md]: contributor, I think?

File: README.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ you would like to participate in the Otrego Community, please read our
 
 ## Developers Guide
 
-If you're interesting in contributing to Otrego as a developer, check out our
+If you are interesting in contributing to Otrego as a developer, check out our
```
Comment (nelsonjhk) [inline README.md:16]: "if you are interest**ed**"

**Review by nelsonjhk — CHANGES_REQUESTED** [review]
A couple of typos and suggestions from Brian and I should be looked at.

File: README.md (hunk 3 above)
Comment (artasparks) [reply README.md]: Used languages instead of the countries.

File: gettingstarted/codereview.md (hunk 4 above)
Comment (artasparks) [reply gettingstarted/codereview.md]: Pull Request. Changed.

File: README.md (hunk 5 above)
Comment (artasparks) [reply README.md:16]: Done.

**Comment by artasparks** [comment]
PTAL, resolved outstanding comments

**Comment by TamaBrian** [comment]
Hi Josh,
   Based on your message I signed into Git    Otrego and browsed changes
you made to some code but I lost my place and can't find it again.   The
code, as I viewed it in SPLIT format, had to do with placing a stone
(color, pt) and detecting if the action actually captured stones or was a
self-capture.   Therein, at the bottom of your changes was a // comment
ending with the letters " th ", without follow on text.   I was going to
comment on it but can't since I got lost.   You can probably find it based
on this.
   And maybe you can show me how to get back to what I viewed.

Thanks,
     Brian   (TamaBrian)

**Pull request merged**